[ADD] pos_check_session_state

legalsylvain opened the pull request:
Add a new simple module, that check that the PoS session is still in a opening state.
It prevents error, if a user close the session in a back office, and another waiter is using one.

**configuration**

![pos_config_form](https://user-images.githubusercontent.com/3407482/76254957-9f218780-624d-11ea-8cb1-5c62b19be509.png)

**Result**

![error_message](https://user-images.githubusercontent.com/3407482/76254954-9d57c400-624d-11ea-9134-1585a9357789.png)

legalsylvain committed:
[ADD] pos_check_session_state

chienandalu reviewed (changes requested):
Nice module :)

Some remarks:

- As the pos window is blocked, it'd be nice to have a button in the dialog to close it:
![image](https://user-images.githubusercontent.com/5040182/63406228-8c4a8100-c3e9-11e9-8fda-aad7ec209428.png)

- Config parameter could be in a more integrated style:
![image](https://user-images.githubusercontent.com/5040182/63406186-6f15b280-c3e9-11e9-970e-b6f5314ab65b.png)

- Check lint errors

legalsylvain committed:
[IMP] various things, cortesy @MeetKD and @chienandalu
[REF] rename files
[UPD] update description images

legalsylvain commented:
Hi all.
Thanks for your review.
@MeetKD : I fixed your points.
@chienandalu : I refactored the code. (more simple, no more new Popup Object), and add a button OK. When clicked the PoS is closed that is the expected behaviour, because it is not possible to use it anymore.

could you update your review ? 

thanks !

legalsylvain committed:
[IMP] pos_check_session_state: make optional the recurrent check, to avoid bad UX in the case of poor connexion to odoo server

MeetKD commented on pos_check_session_state/static/src/js/chrome.js:
```diff
@@ -0,0 +1,71 @@
+            });
+        }
+    });
+
```
Can we remove extra line ?

MeetKD commented on pos_check_session_state/views/view_pos_config.xml:
```diff
@@ -0,0 +1,31 @@
+                        </div>
+                    </div>
+                </div>
+
```
Can we remove extra line here too ?

legalsylvain commented:
@ MeetKD said:

> Can we remove extra line ?

I personally don't see why. Matter of taste.... But sure, feel free to make a PR, if you think it's important !

In the meantime, could you update your review ? 

Thanks !

MeetKD reviewed (approved):
@legalsylvain Yes its matter of taste :+1:  Updated Review for now.

legalsylvain commented:
Hi @rafaelbn, could you unblock this PR ? I think i made the required changes.

thanks !

chienandalu commented:
Sorry for the late review

/ocabot merge nobump
